pandas-stubs/core/generic.pyi

```diff
@@ -386,7 +387,7 @@ class NDFrame(PandasObject, indexing.IndexingMixin):
         inplace: _bool = ...,
         limit=...,
         regex: _bool = ...,
-        method: _str = ...,
+        method: Literal["pad", "ffill", "bfill"] | None = ...,
```

Comment:
Don't think that `None` is valid

Reply:
Docs say it is accepted: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.replace.html

Reply:
I think the docs are wrong.  
```python
>>> s=pd.Series([3,4,5])
>>> s.replace(None, method=None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Anaconda3\envs\pandasstubs\lib\site-packages\pandas\core\series.py", line 4960, in replace
    return super().replace(
  File "C:\Anaconda3\envs\pandasstubs\lib\site-packages\pandas\core\generic.py", line 6615, in replace
    return self._replace_single(to_replace, method, inplace, limit)
  File "C:\Anaconda3\envs\pandasstubs\lib\site-packages\pandas\core\series.py", line 5000, in _replace_single
    fill_f = missing.get_fill_func(method)
  File "C:\Anaconda3\envs\pandasstubs\lib\site-packages\pandas\core\missing.py", line 907, in get_fill_func
    return _fill_methods[method]
KeyError: None
```
I looked in the code, and we don't accept `None` in the typing for `replace()`.  

So can you take `None` out of here, and open a PR in pandas to fix the docs?

Reply:
I guess `None` worked at some point and then the implementation switched to `NoDefault.no_default` and removed support for `None`.

Reply:
None is already removed from the development docs :)

https://pandas.pydata.org/docs/dev/reference/api/pandas.Series.replace.html